src/Containers/packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -216,11 +217,69 @@
                     ExposedPorts="@(ContainerPort)"
                     ContainerEnvironmentVariables="@(ContainerEnvironmentVariables)"
                     ContainerRuntimeIdentifier="$(ContainerRuntimeIdentifier)"
-                    ContainerUser="$(ContainerUser)"
-                    RuntimeIdentifierGraphPath="$(RuntimeIdentifierGraphPath)"> <!-- The RID graph path is provided as a property by the SDK. -->
+                    ContainerUser="$(ContainerUser)">
 
       <Output TaskParameter="GeneratedContainerManifest" PropertyName="GeneratedContainerManifest" />
       <Output TaskParameter="GeneratedContainerConfiguration" PropertyName="GeneratedContainerConfiguration" />
     </CreateNewImage>
   </Target>
+
+  <!-- Secondary entrypoint, either from solution-level `/t:Containerize` or project-level `/t:Containerize` -->
+  <Target Name="Containerize" Condition="'$(EnableSdkContainerSupport)' == 'true'">
+    <PropertyGroup>
+      <!-- We have to build Publish AND PublishContainer because PublishContainer (and other
+          PublishProfile-delivered targets) don't have an explicit Publish dependency. -->
+      <_RequiredContainerPublishTargets>Publish;PublishContainer</_RequiredContainerPublishTargets>
+    </PropertyGroup>
+
+    <!-- Strategy here is that we will figure out what project(s) to build the containerization targets(s) for
+         based on project state. We use `AdditionalProperties` to customize the outputs of each of the builds. -->
+
+    <!-- Properties set here:
+      * TargetFramework - multitargeting - changes inference for base image based on TFM
+      * ContainerImageTag - this is a rough cut
+      * ContainerRuntimeIdentifier - if we're building for a specific RID, we need to set this so that the
+                                     containerization targets know what RID to build for
+      * RuntimeIdentifier - if we're building for a specific RID, we need to set this so that we get optimized
+                            RID-specific assets in the publish output
+
+      NOTE: we could get away with setting `RuntimeIdentfier` here to control `ContainerRuntimeIdentifier` inference
+            but this is also nice and explicit.
+     -->
+
+    <!-- TFMs but no TF -> multitarget, making image for each TFM -->
+    <ItemGroup Condition="'$(TargetFrameworks)' != ''
+                          and '$(TargetFramework)' == ''" >
+      <_TFMItems Include="$(TargetFrameworks)" />
+      <_SingleContainerPublish Include="$(MSBuildProjectFullPath)"
+        AdditionalProperties="TargetFramework=%(_TFMItems.Identity);
+                              ContainerImageTag=$([MSBuild]::GetTargetFrameworkVersion('%(_TFMItems.Identity)', 2))" />
```

Comment:
Discussion: we need to disambiguate tags here because version/latest doesn't make unique container/tag pairs. I've gone with TFMs here. Is this useful, or should we sidestep/ignore/error the multi-TFM containerization case?